netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1183,7 +1170,7 @@ net_ebpf_extension_sock_addr_authorize_connection_classify(
     }
     verdict = connection_context->value.verdict;
 
-    _net_ebpf_ext_delete_connection_context(connection_context);
+    ExFreePool(connection_context);
```

Comment:
I am assuming ExFeePool does NUL check?

Reply:
`ExFreePool` is called only when `_net_ebpf_ext_get_and_remove_connection_context()` succeeds. So NULL check is not needed.